src/models/IRoutePath.ts

```diff
@@ -12,4 +12,8 @@ export default interface IRoutePath {
     endTime: Date;
     lastModified: Date;
     routePathLinks: IRoutePathLink[];
+    originFi: string;
+    originSe: string;
+    destinationFi: string;
+    destinationSe: string;
```

Comment:
destinationSe and originSe not being used. Why do we query them in if we don't use them?

Reply:
I like swedish

Reply:
I'm kidding, I'll remove it